shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/ApiArrivalsWithSplits.scala

```diff
@@ -40,19 +40,19 @@ case class ApiFlightWithSplits(apiFlight: Arrival, splits: Set[Splits], lastUpda
 
   def totalPax: Option[TotalPaxSource] =
     if (hasValidApi) totalPaxFromApi
-    else bestSource(apiFlight.ActPax)
+    else bestSource
 
   def equals(candidate: ApiFlightWithSplits): Boolean =
     this.copy(lastUpdated = None) == candidate.copy(lastUpdated = None)
 
-  def bestSource(actPax: Option[Int]): Option[TotalPaxSource] = {
+  def bestSource: Option[TotalPaxSource] = {
```

Comment:
I'm thinking we probably don't need this function. It's only used in one place, and it doesn't use all the available sources. We should probably drop it in favour of a new function `Arrival.bestPaxEstimate` which should behave the same way as `Arrival.bestPcpPaxEstimate` but return `Option[Passengers]`. `bestPcpPaxEstimate` can then just use that one and return the calculated pcp pax